src/tests/functional/plugin/shared/src/low_precision_transformations/convolution_transformation.cpp

```diff
@@ -36,9 +36,6 @@ std::string ConvolutionTransformation::getTestCaseName(const testing::TestParamI
 }
 
 void ConvolutionTransformation::SetUp() {
-    rel_threshold = 1.0e+10;
```

Comment:
can you explore this case in more details? I think something wrong here.

Reply:
my mistake

Reply:
resolved